src/OSPSuite.Core/Domain/Services/IHistoryManagerRetriever.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OSPSuite.Core.Commands.Core;
```

Comment:
This service will have to be implemented in both apps, as the HistoryManager is not stored in the same place